subcommands/config/updates.go

```diff
@@ -20,11 +20,31 @@ in a device group. When run without options, prints out the current configuratio
   # Make devices start taking updates from Targets tagged with "devel":
   fioctl config updates --group beta --tag devel
 
-  # Set the docker apps that devices will run:
+  # Set the Compose apps that devices will run:
   fioctl config updates --group beta --apps shellhttpd
 
-  # Set the docker apps and the tag for devices:
-  fioctl config updates --group beta --apps shellhttpd --tag master`,
+  # Set the Compose apps and the tag for devices:
+  fioctl config updates --group beta --apps shellhttpd --tag master
+
+	# There are two special characters: "," and "-"
+	# The following explains what happens if you provide a ","
+  # Set the Compose apps to none for devices, this will make the device run no apps:
+  fioctl config updates --apps ,
+
+  # The following explains what happens if you provide a "-"
+	# Set the Compose apps to preset-apps (all apps on most devices):
+	# 	The system will look in the following locations in order to get the complete config:
+       - /usr/lib/sota/conf.d/
+       - /var/sota/sota.toml
+       - /etc/sota/conf.d/
+  fioctl config updates --apps -
+
+  # Set the device tag to a preset-tag :
+	# 	The system will look in the following locations in order to get the complete config:
+       - /usr/lib/sota/conf.d/
+       - /var/sota/sota.toml
+       - /etc/sota/conf.d/
+  fioctl config updates --tag -`,
```

Comment:
An indentation here seems rather weird.
Maybe, @kprosise can help with it?

At the least, these two sentences should be indented at the same level:
```
# The following explains what happens if you provide a ","
# The following explains what happens if you provide a "-"
```

The same applies to these three sentences (about indentation):
```
# Set the Compose apps to none for devices, this will make the device run no apps:
# Set the Compose apps to preset-apps (all apps on most devices):
# Set the device tag to a preset-tag :
```

Also, as sentences in these paragraphs are user-facing, they deserve a proper punctuation at their ends, like `.` or `:` (and proper spacing, like e.g. no space before a colon).